src/components/Dashboard/TodoList.vue

```diff
@@ -543,7 +543,9 @@ export default {
           filingDate: filing.correction.correctedFilingDate,
           corrFilingId: filing.correction.correctedFilingId,
           correctedFilingType: this.typeToTitle(filing.correction.correctedFilingType),
-          title: `Priority Correction - ${this.typeToTitle(filing.correction.correctedFilingType)}`,
+          title: `${this.isPriority(filing.header.priority)} -
```

Comment:
This still displays the " - " if this is not a priority filing right? Could you clean this up please?